Agreements/Internal/TCWEL_28.txt

```diff
@@ -0,0 +1,42 @@
+COPX:EGKK:*:BEDFO:*:*:London TC WELIN:London S28:21000:*:^BEDFO
```

Comment:
Should we have an EGTK departure agreement? 190 SAPCO

Reply:
Added